app/src/services/cpk.ts

```diff
@@ -130,12 +131,20 @@ class CPKService {
     return this.cpk.address
   }
 
-  waitForTransaction = async (txObject: TransactionResult): Promise<TransactionReceipt> => {
+  waitForTransaction = async (txObject: TransactionResult, xDaiBridge?: boolean): Promise<TransactionReceipt> => {
```

Comment:
This works just fine! Potentially this could be a `confirmations` param instead of `xDaiBridge` if we want to make this more generic.